x-pack/test/case_api_integration/common/fixtures/plugins/cases_client_user/kibana.json

```diff
@@ -1,10 +1,11 @@
 {
   "id": "casesClientUserFixture",
+  "owner": { "name": "Security solution", "githubTeam": "security-solution" },
```

Comment:
Could we switch this to the threat hunting team? We're the ones that manage these tests:

```
  "owner":{
     "githubTeam":"security-threat-hunting",
     "name":"Security Solution Threat Hunting"
  },
```